bindings/csharp/Import.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Keystone
+{
+    class Import
+    {
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern int ks_version(ref int major, ref int minor);
+
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern bool ks_arch_supported(Constants.ks_arch arch);
+
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern Constants.ks_err ks_open(Constants.ks_arch arch, int mode, ref IntPtr ks);
+
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern Constants.ks_err ks_close(IntPtr ks);
+
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern Constants.ks_err ks_errno(IntPtr ks);
+
+        [DllImport("keystone.dll", CallingConvention = CallingConvention.Cdecl)]
+        public static extern string ks_strerror(Constants.ks_err code);
```

Comment:
I guess this won't work. 
This function is not covered by your TestRunner and in fact if I try to call it explicitely it make my process crash. See http://stackoverflow.com/questions/6300093/why-cant-i-return-a-char-string-from-c-to-c-sharp-in-a-release-build